src/nvim/if_cscope.c

```diff
@@ -2073,14 +2073,13 @@ static char *cs_resolve_file(int i, char *name)
  */
 static int cs_show(exarg_T *eap)
 {
-  short i;
   if (cs_cnt_connections() == 0)
     MSG_PUTS(_("no cscope connections\n"));
   else {
     MSG_PUTS_ATTR(
         _(" # pid    database name                       prepend path\n"),
         hl_attr(HLF_T));
-    for (i = 0; i < csinfo_size; i++) {
+    for (size_t i = 0; i < csinfo_size; i++) {
```

Comment:
The value of i is used in format strings later. You changed the type from signed to an unsigned integral number, but the format strings expect a signed one.